### ISSUE TITLE
Control File: Using M with L - is it possible?

### ISSUE THREAD

**Issue by pobtastic** [skoolkid/skoolkit#128]
Maybe just an expectation thing?  But using `M` which I'd assumed to be on the sub-block level together with `L` like this:
```
g $C1A1 Table: Sprite Information
@ $C1A1 label=Table_SpriteInfo
N $C1A1 #LET(sprite=$01+(#PC-$C1A1)/$04) Sprite ID: #N({sprite}).
M $C1A1,$08 #SPRITE({sprite})
B $C1A1,$02
W $C1A3,$02
B $C1A5,$02
W $C1A7,$02
L $C1A1,$08,$70
```
...doesn't seem to work.  It's not a huge deal, I could just script it - I just wondered if I'm overlooking something obvious here?

**Comment by skoolkid** [comment]
No, I'd expect `L` to repeat `M` as well. I'll look into it.

**Comment by skoolkid** [comment]
Fixed in e96cf46546da78bd86b23498bf357ba801e241a8. Let me know if it works as expected now.

**Comment by pobtastic** [comment]
Hiya!  Excellent, thank you so much!  That works great :)

<img width="612" alt="Screenshot 2024-02-01 at 08 06 47" src="https://github.com/skoolkid/skoolkit/assets/1250035/b02d7ade-a5ad-48e8-abfb-11b531109486">
